### PR TITLE
Replaced ZeroClipboard with clipboard | fixes #313

### LINKED ISSUE #313: use clipboard.js instead of zero-clipboard
we should use the new clipboard.js[1] library which uses more modern browser apis to implement copy&paste without flash.

its the successor of the lib we are using right now [2].
Atm whoops loads a SWF file when rendered to be able to provide the copy&paste functionality.

[1] https://github.com/zenorocha/clipboard.js
[2] https://github.com/zeroclipboard/zeroclipboard

only disadvantage I see right now is, that Safari doesnt implement some of the APIs required for the new lib, see https://github.com/zenorocha/clipboard.js#browser-support

### PR DESCRIPTION
**Pull request by prisis**

### CODE REVIEW

**Comment by denis-sokolov** [comment]
Seems good to me. @staabm, do you see any problems?

File: src/Whoops/Resources/css/whoops.base.css [hunk 1]
```diff
@@ -469,3 +468,58 @@ kbd {
...
+  right: 50%;
+  margin-top: 5px
+}
+.tooltipped-s:before {
+  top: auto;
+  right: 50%;
+  bottom: -5px;
+  margin-right: -5px;
+  border-bottom-color: rgba(0, 0, 0, 0.8)
+}
```
Comment (GrahamCampbell) [inline src/Whoops/Resources/css/whoops.base.css]: missing newline

Comment (prisis) [inline src/Whoops/Resources/css/whoops.base.css]: done.

**Comment by staabm** [comment]
Looks good, didnt test it though

**Pull request merged**